src/index.ts

```diff
@@ -0,0 +1,357 @@
+  {
+    name: 'verify report',
+    type: String,
+    typeLabel: '{italic [--format] uuid}',
+    description: 'Get the report of a completed analysis.'
+  },
+  {
+    name: 'verify status',
+    type: String,
+    typeLabel: '{italic uuid}',
+    description: 'Get the status of an already submitted analysis.'
+  },
+  {
+    name: 'verify list',
+    description: 'Displays a list of the last 20 submitted analyses in a table.'
+  },
+  {
+    name: 'verify help',
+    type: Boolean,
+    defaultValue: false,
+    description: 'Display this usage guide.'
+  }
+];
+
+const header =
+  'Smart contract security analysis with MythX\n\n' +
+  // tslint:disable: no-trailing-whitespace
+  chalk.blueBright(`         ::::::: \`          :::::::\`      \`\`\`   \`\`\`                     \`\`        \`\`     \`\`\`         
+         +++++++\`          +++++++\`      ...\` \`...              .\`     ..        \`.\`   \`.\`          
+         \`\`\`:+++///:   -///+++/\`\`\`       ..\`. .\`..  \`\`    \`\`  \`\`..\`\`   ..\`\`\`\`     \`.. \`.\`           
+            -++++++/   :++++++:          .. .\`. ..  \`.\`   .\`   \`.\`     ..\` \`..      ...             
+                /++/   :+++              .. ..\` ..   ..  ..     .\`     ..   \`.     \`...\`            
+            \`\`\`\`////\`\`\`:///.\`\`\`          ..     ..    .\` .\`     .\`     ..   \`.    \`.\` \`.\`           
+            -+++\`  \`+++-   +++:          ..     ..    \`...      ..\`\`   ..   \`.   \`.\`   \`..          
+            .:::\`  \`:::.   :::.          \`\`     \`\`     ..\`      \`\`\`\`   \`\`   \`\`  \`\`      \`\`\`         
+                                                     \`\`..                                           
+                                                      \`                                             
+`);
+// tslint:enable: no-trailing-whitespace
+
+const CLI_USAGE = [
+  {
+    header: 'embark-mythx',
+    content: header,
+    raw: true
+  },
+  {
+    header: 'Available Commands',
+    content: Array.from(new Set(CLI_COMMANDS.values())).map(command => {
+      return {
+        name: `${command.name} ${command.typeLabel || ''}`,
+        summary: command.description
+      };
+    })
+  },
+  {
+    header: 'Examples',
+    content: [
+      {
+        name: 'verify --mode full SimpleStorage ERC20',
+        summary:
+          'Runs a full MythX verification for the SimpleStorage and ERC20 contracts only.'
+      },
+      {
+        name: 'verify status 0d60d6b3-e226-4192-b9c6-66b45eca3746',
+        summary:
+          'Gets the status of the MythX analysis with the specified uuid.'
+      },
+      {
+        name:
+          'verify report --format stylish 0d60d6b3-e226-4192-b9c6-66b45eca3746',
+        summary:
+          'Gets the status of the MythX analysis with the specified uuid.'
+      }
+    ]
+  },
+  {
+    header: 'Verify options',
+    hide: ['contracts'],
+    optionList: CLI_OPTS,
+    group: ['options']
+  }
+];
```

Comment:
All the CLI stuff could be put in another file, so that the index stays cleaner

Reply:
Moved to `./cli`.